initializers/smart-agent-core.js

```diff
@@ -36,6 +37,13 @@ module.exports = class SmartAgentCore extends Initializer {
       registeredAgents: []
     }
 
+    // start the initialization code
+    const smartAgentCore = new SmartAgent(api.config.smartAgentCore)
+    await smartAgentCore.initialize()
+
+    // objects and values used outside initializer
+    api.smartAgentCore = smartAgentCore
+
```

Comment:
(belongs to question group 1)
❓ Does the seed need its own runtime? Derived smart agents will have a running runtime and the seed itself will most probably never use its own `runtime` and auth. So every Serve would have an extra runtime with default accounts and a key exchange listener, that does nothing or am I wrong?
imho we could move the `_ensureAuth` into the class scope and use the `runtime` from the current instance... or? ^^
```suggestion
```